website/templates/website/password_reset_email.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n %}{% autoescape off %}
```

Comment:
why is this in this pull request?

Reply:
The email template was included because the link's didn't include the website namespace resulting in a url error when Django tried to send emails.

Reply:
i think this file needs removed